client-cli/src/command/wallet_command.rs

```diff
@@ -31,6 +35,35 @@ pub enum WalletCommand {
         )]
         wallet_type: WalletKind,
     },
+    #[structopt(name = "export", about = "Backup wallet to a file")]
+    Export {
+        #[structopt(
+            name = "wallet name",
+            short = "n",
+            long = "name",
+            help = "Name of wallet"
+        )]
+        name: String,
+        #[structopt(
+            name = "file",
+            short = "f",
+            long = "file",
+            parse(from_os_str),
+            help = "file to dump wallet"
+        )]
+        file: PathBuf,
+    },
+    #[structopt(name = "import", about = "Import a wallet from a file")]
+    Import {
+        #[structopt(
+            name = "file",
+            short = "f",
+            long = "file",
+            parse(from_os_str),
+            help = "file stored the wallet info"
+        )]
+        file: PathBuf,
```

Comment:
I suggest to treat a special value "-" as stdin and make it the default value, so the command can work with pipes in shell.

Reply:
Currently, I serialized the wallet info as multiline json string using the `serde_json::to_string_pretty`. 

If inputted from the terminal, I have to serialize the wallet info into one line json string. By the way, the output is colored and there is a waring messge like `Warning! `CRYPTO_CHAIN_ID` environment variable is not set. Setting network to devnet and network-id to 0`. I don't think the the pipes can work well.

Reply:
I mean something like this:
```
let output = if name == "-" {
  std::io::stdout
} else {
  File::open(name)...
};
output.write_all(...);
```

Reply:
- is too short , "console"?

Reply:
> `-` is too short , "console"?

`-` is like a conventional name for this.

Reply:
This is one way to use it, but only some developers know it.

Reply:
> How does the command work with `-`, like this?
> 
> ```
> cat wallet_file | ./client-cli wallet import -
> ```

Yes, with pipes it can work with other tools together, for example, compression tool:
```
$ ./client-cli wallet export --all | gzip > wallets.export.gz
$ zcat wallets.export.gz | ./client-cli wallet import
```